public/stock-controller.js

```diff
@@ -1,12 +1,12 @@
 (function() {
     'use strict';
 
-    angular.module('openfin.stock', ['openfin.quandl'])
-       .controller('StockCtrl', ['$routeParams', 'quandlService', function($routeParams, quandlService) {
+    angular.module('openfin.stock', ['openfin.quandl', 'openfin.openfin'])
+       .controller('StockCtrl', ['$routeParams', 'quandlService', 'openfinService', function($routeParams, quandl, openfin) {
```

Comment:
As in the name of the input parameters in `function( ... )` should be the same as the injected names before in the array?